src/WebJobs.Script/Extensions/FileUtility.cs

```diff
@@ -246,8 +246,14 @@ public static void DeleteDirectoryContentsSafe(string path, bool ignoreErrors =
 
         public static void DeleteFileSafe(string path)
         {
-            var info = FileInfoFromFileName(path);
-            DeleteFileSystemInfo(info, ignoreErrors: true);
+            try
```

Comment:
Was this not that safe before? :)

Reply:
No it wasn't - I was attempting to use FileUtility.DeleteFileSafe in my GetRoutePrefix_Succeeds test and I was getting exceptions bubbling out. Even with this change I was getting concurrency issues. I suspect that we have an issue with FileUtility.ReadAsync which I was using in the product code to read host.json. I switched to File.ReadAllText and the problems went away.